osp17_ref/overcloud_deploy_regular.sh

```diff
@@ -0,0 +1,21 @@
+echo "Creating roles..."
```

Comment:
We have to figure out why using dpdk and sriov env are trigger unprovision, IMO, those envs are required. other than it, templates looks fine to me.